internal/lbapi/types.go

```diff
@@ -0,0 +1,29 @@
+package lbapi
+
+type Origin struct {
+	ID        string `json:"id"`
+	Name      string `json:"name"`
+	IPAddress string `json:"origin_target"`
+	Disabled  bool   `json:"origin_disabled"`
+	Port      int64  `json:"port"`
+}
+
+type Pool struct {
+	ID      string   `json:"id"`
+	Name    string   `json:"name"`
+	Origins []Origin `json:"origins"`
+}
+
+type Port struct {
+	AddressFamily string   `json:"address_family"`
+	ID            string   `json:"id"`
+	Name          string   `json:"name"`
+	Port          int64    `json:"port"`
+	Pools         []string `json:"pools"`
+	PoolData      []Pool
```

Comment:
```suggestion
	PoolData      []Pool `json:"_"`
```

you shouldnt allow json data into this field i think?

Reply:
yah, since that client will eventually live in the lbapi code base, I think I'm going to remove `PoolData` from the struct and just pass `Pools` to that merge function.